### PR TITLE
Initialize jsonnetfmt action

### PR DESCRIPTION
**Pull request by kakkoyun**
Let there be light!

### CODE REVIEW

**Comment by kakkoyun** [comment]
cc @metalmatze @aweris

**Comment by kakkoyun** [comment]
I haven't tested it yet 😅

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+
+COPY --from=builder /go/bin/jsonnetfmt /bin/jsonnetfmt
+
+LABEL repository="https://github.com/kakkoyun/action-jsonnetfmt"
+LABEL homepage="https://github.com/kakkoyun/action-jsonnetfmt"
+LABEL maintainer="Kemal Akkoyun <kakkoyun@gmail.com>"
+
+LABEL "com.github.actions.name"="jsonnetfmt Action"
+LABEL "com.github.actions.description"="Run jsonnetfmt to check if jsonnet files are formatted in current workspace"
+LABEL "com.github.actions.icon"="jsonnet"
```
Comment (kakkoyun) [inline Dockerfile]: I'm not sure what this is YOLO

Comment (aweris) [reply Dockerfile]: This is same as 
```yml
name: 'Hello World'
description: 'Greet someone and record the time'
```

You're using `action.yml` no need to ass these labels.

Comment (kakkoyun) [reply Dockerfile]: What about `actions.icon`?

Comment (aweris) [reply Dockerfile]: You should add it to `action.yml`

**Pull request merged**